src/Main.java

```diff
@@ -2,6 +2,149 @@
 
 public class Main {
     public static void main(String[] args) {
+
+        int salary = 15000;
+        int total = 0;
+        int i = 0;
+        while (total <= 2_459_000) {
+            total = total + salary;
+            total = total + total / 100;
+            i = i + 1;
+            System.out.println(" Месяц " + i + " сумма накоплений равна " + total + " рублей ");
+        }
+        System.out.println();
+        int a = 0;
+        while (a < 10) {
+            a = a + 1;
+            System.out.print(" " + a);
+        }
+        System.out.println();
+        int start = 11;
+        for (; start > 1; ) {
+            start = start - 1;
+            System.out.print(" " + start);
+        }
+        System.out.println();
+        System.out.println();
+        int years = 10;
+        int population = 12_000_000;
+        int birthRate = population * 17 / 1000;
+        int deathRate = population * 8 / 1000;
+        for (; i < 10; ++i) {
```

Comment:
```suggestion
        for (int year = 1; year <= years; year++) {
```
используем переменные